lagtraj/trajectory/create.py

```diff
@@ -278,6 +278,7 @@ def _calculate_seconds(da):
         else:
             raise Exception
 
+        print("Creating " + dir + " part of trajectory")
```

Comment:
I think you have a little debug `print` here :)

Reply:
Should have made this a separate request maybe. I like having this in, as it makes it explicit that the status bars only refer to part of the trajectory, rather than the whole trajectory (this can be confusing when you see the status bar come up and it has an unexpected number of steps)

Reply:
Ah, good idea! Ok, `tqdm` has a `desc` attribute (https://github.com/tqdm/tqdm#parameters) that works quite nicely for this. So the line below would just need changing
```python
for t in tqdm.tqdm(da_integrate_times)
```
to
```python
for t in tqdm.tqdm(da_integrate_times, desc=dir)
```
sound ok?

Reply:
Yes, great suggestion!